plugin.video.ytchannels/addon.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.ytchannels" name="YouTube Channels" version="0.3.6+matrix.1" provider-name="natko1412,mintsoft,yeahme49">
+<addon id="plugin.video.ytchannels" name="YouTube Channels" version="0.4.0+matrix.1" provider-name="natko1412,mintsoft,yeahme49">
 	<requires>
 		<import addon="xbmc.python" version="3.0.0"/>
 		<import addon="script.module.six" version="1.11.0"/>
 		<import addon="plugin.video.youtube"/>
+		<import addon="plugin.video.invidious" optional="true"/>
 	</requires>
```

Comment:
What is plugin invidious? We don't have this on our repo..

Reply:
It is this: https://github.com/TheAssassin/kodi-invidious-plugin

It's essentially a privacy proxy for Youtube, it's an optional feature and the settings are only visible if the plugin is enabled already.